app/assets/stylesheets/_screen.scss

```diff
@@ -625,6 +617,63 @@ div#keycut-help {
   }
 }
 
+/* Settings */
+.settings-page {
+  height: 100%;
+  padding: 0 30px;
+  overflow-y: auto;
+}
+
+/* Import */
+.import-form {
```

Comment:
Does it belong to settings page? If so, it should also be inside `_settings.scss`. I also think `.import-form` name is too generic, what does it import? Can I use it for any form where I want to import something?